schemas/mozilla_nimbus_schemas/experiments/feature_manifests.py

```diff
@@ -0,0 +1,139 @@
+from enum import Enum
+from typing import Literal, Optional, Union
+
+from pydantic import BaseModel, Extra, Field, create_model
+
+
+class FeatureVariableType(Enum):
+    INT = "int"
+    STRING = "string"
+    BOOLEAN = "boolean"
+    JSON = "json"
+
+
+FEATURE_SCHEMA_TYPES = {
+    FeatureVariableType.INT: "integer",
+    FeatureVariableType.STRING: "string",
+    FeatureVariableType.BOOLEAN: "boolean",
+}
```

Comment:
What do these do?

Reply:
Oops, don't need these. These were copy+pasted from the other python module.